e2e/cypress/integration/api/iam/04_projects_api_client_runs_ingestion.ts

```diff
@@ -168,28 +169,14 @@ describeIfIAMV2p1('Client Runs Ingestion project tagging', () => {
       url: '/apis/iam/v2beta/apply-rules'
     });
 
-    waitUntilApplyRulesNotRunning(100);
+    cy.waitUntilApplyRulesNotRunning(100);
```

Comment:
Out of curiosity, what influences the argument for this?

Reply:
The 100 here means we will wait 100 seconds for the "apply rules" to complete. It has normally taken less than 30 seconds.

Reply:
I understand what it _does_ . . . I'm just curious what makes us choose `100` or `500` or what would make us need to change it.

Reply:
The reason I choose 100 is that, if it normally takes about 30 seconds then if it takes 100 we would want to stop the test. I could see maybe using 60, but sometimes buildkite gets slowed down and it might take longer to complete.